res/values/strings.xml

```diff
@@ -1621,9 +1608,9 @@
     <string name="reminder_header_push_title">Enable Signal messages and calls</string>
     <string name="reminder_header_push_text">Upgrade your communication experience.</string>
     <string name="reminder_header_invite_title">Invite to Signal</string>
-    <string name="reminder_header_invite_text">Take your conversation with %1$s to the next level.</string>
+    <string name="reminder_header_invite_text">Suggest to %1$s to continue this conversation as a secure Signal conversation.</string>
     <string name="reminder_header_share_title">Invite your friends!</string>
-    <string name="reminder_header_share_text">The more friends use Signal, the better it gets.</string>
+    <string name="reminder_header_share_text">The more people use Signal, the more often you can communicate in private.</string>
```

Comment:
This is not used in Molly. I cannot find where it is used in Signal either.

Reply:
I remember seeing it in the app, it was a text on one of those dismissable header pop-ups. I think it is shown when you communicate with people who are not using signal yet.